src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1182,6 +1182,13 @@ def get(self, zonegroup_name):
         result = multisite_instance.get_zonegroup(zonegroup_name)
         return result
 
+    @Endpoint('GET')
+    @ReadPermission
+    def get_placement_target_by_placement_id(self, placement_id):
```

Comment:
always expose a nice and readable path for the API. Also you can keep the def name shorter and properly type the variable
```suggestion
    @Endpoint('GET', path='placement-target')
    @ReadPermission
    def get_placement_target(self, placement_id: str):
```